### DISCUSSION TITLE
Add zellij display mode

### DISCUSSION THREAD

**Discussion by noahbald** [junegunn/fzf#4134] [Ideas]
It would be nice if an option similar to `fzf --tmux` was added for zellij, to display a floating pane for those using zellij instead of tmux

**Comment by junegunn** [comment]
I like the idea. It would be nice if fzf could automatically choose tmux popup or zellij floating pane when generic `--popup` option is given.

It would be trivial to implement it if `zellij run` provided an option to block until completion similarly to `tmux display-popup`, but unfortunately it doesn't (https://github.com/zellij-org/zellij/issues/3018), so we would need another mechanism (such as using named pipes).

**Reply by junegunn** [reply]
A PoC implementation: #4145

**Reply by noahbald** [reply]
Wow that’s amazing, I’ll see if I can get a chance to build and play with it
